src/components/Footer.js

```diff
@@ -1,17 +1,18 @@
-import data from "@/config/app.json";
+import data from '@/config/app.json';
```

Comment:
This should remain as double quotes to be consistent with our project `"`, please check you are using our prettier settings

Reply:
I will fix in another comment